packages/wdio-cucumber-framework/src/cucumberEventListener.ts

```diff
@@ -13,10 +13,12 @@ import type { Capabilities } from '@wdio/types'
 import { addKeywordToStep, filterPickles, getRule } from './utils.js'
 import type { ReporterScenario } from './constants.js'
 import type { HookParams } from './types.js'
+import path from 'path'
 
 const log = logger('CucumberEventListener')
 
 export default class CucumberEventListener extends EventEmitter {
+    private _cwd: string
```

Comment:
```suggestion
    #cwd = process.cwd()
```

Reply:
Why remove `cwd` here? We need to pass it in to get the relative path of the pickle URI.

Reply:
`cwd` is not removed, I would just prefer to define the variable as JS private rather then TS private variable, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
Never knew this was a thing. awesome